vivo-project/VIVO-languages
# [i18n redesign] adding prefix to theme property files names (vivo is the prefix)

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3760](https://github.com/vivo-project/VIVO/issues/3760)

# What does this pull request do?
Enabling this [PR](https://github.com/vivo-project/Vitro/pull/341) to have appropriate context in transferring property files into graph. Namely, hasApp data property in the graph should point whether some property key is from VIVO or Vitro application. The previously mentioned PR is loading property files from [tomcat]/webapps/vivo/themes/..., where are overlaid property files from Vitro and VIVO, therefore appropriate prefix in the name of the file is necessary.   

# What's new?
Adding a prefix (vivo) to all property files in VIVO-languages. Previously, it was missed in the theme property files. 

# How should this be tested?
Check the names of theme property files came from VIVO/VIVO-language project, it should start with vivo_all_*

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
